ocaml/xapi/xapi_globs.ml

```diff
@@ -846,6 +846,8 @@ let varstore_rm = ref "/usr/bin/varstore-rm"
 
 let varstore_dir = ref "/usr/share/varstored"
 
+let fallback_auth_dir = ref "/var/lib/varstored"
```

Comment:
How about exchanging the default locations of `varstore_dir` and `fallback_auth_dir` (maybe rename it `varstore_fallback_dir` in order to share a common prefix with `varstore_dir`?).

I know this would temporarily break varstored on Citrix Hypervisor, but it would be easily fixed in `xapi.conf` or `xapi.conf.d` and allow to be consistent right from the start in the code: `varstore_dir`, volatile, in `/var`, and fallback_dir in `/usr`.

Reply:
I think it's for @edwintorok to decide.

Reply:
@psafont @edwintorok What do you think about my comment here? (for a possible future PR)